design/src/high-level/analysis-graphics.md

```diff
@@ -0,0 +1,101 @@
+# High-Level Analysis: Graphics
+
+## Case Studies
+
+### Terra Nil
+
+[A reverse civilization builder solar punk game](https://store.steampowered.com/app/1593030/Terra_Nil/) where the player is tasked with making a barren planet bustling with life. The player builds machines to purify water, soil, and air to breathe life back into the game world.
+
+- A _2D world_ with an _isometric camera with constrained motion_.
+
+- Animals and some player buildings' _2D assets_ have _outlines drawn with, and cell shading using, shaders_ to add definition to assets.
+
+- Various animals have _'random' animations_ that add a sense of life to the game.  
+
+- The color palette is _vivid colors for greenery_, _muted colors for wasteland_, and _soft hues of bold colors for buildings_ which helps fit the game's theme of _rejuvenating nature_.
+
+- _Textures_ give the assets _varying levels of depth_.
+
+- _Ambient lighting_ gives depth to the world, _casts shadows_, and helps _ambient clouds add texture/depth_.
+
+- Assets have a _pastoral, hand drawn aesthetic_ with mild _cell shading_.
+
+### Warcraft 3
+
+[A high-fantasy RTS game](https://en.wikipedia.org/wiki/Warcraft_III:_Reign_of_Chaos) where the player is tasked with defending the realm from an impending take-over by the Burning Legion.
+
+- _3D world_ with a _fixed angle, isometric camera_.
+
+- _Fog of war_ hides the map until the player explores it, a common feature of RTS games.
+
+- Heavy use of _baked, ambient light_ for the world and _soft shadows_ for units.
```

Comment:
Oh, this is pretty neat! 

Can we add some relevant links for things like "baked, ambient light" and "soft shadows"? If not part of this PR, that's okay, we can make an issue for this?

Reply:
This belongs in the glossary :) Probably a seperate rendering section?